modules/primer-navigation/README.md

```diff
@@ -93,12 +93,123 @@ You can also add optional headings to a menu. Feel free to use nearly any semant
 </nav>
 ```
 
+## Underline nav
+
+`.UnderlineNav` is navigation that is typically used at the top of a page as the main page navigation.
```

Comment:
This could be worded better, it says navigation twice and descriptions could be a little more helpful. Suggest something like this but feel free to edit:

> Use `UnderlineNav` to style navigation with a minimal underlined selected state. This component comes with variations to accommodate icons, counters, and button actions; and is typically used for navigation placed at the top of the page.